### DISCUSSION TITLE
Slow speed on Asus RT-AC88U

### DISCUSSION THREAD

**Discussion by twatwak** [fakemanhk/openwrt-jp-ipoe#35] [Q&A] [answered]
Recently flashed openwrt 23.05 on Asus RT-AC88U and setup ipoe following this guide.
Browsing works fine but my average speed got decreased to around 270-290Mbps DL and 300Mbps UL.
My previous router TP-Link Archer A72 gives around 500-600Mbps DL/UL.
Any ideas or suggestions? Any help is appreciated.

**Comment by fakemanhk** [comment, marked as answer]
Asus RT-AC88U is Broadcom SoC, without proprietary firmware/driver there
must be performance hit.

**Reply by twatwak** [reply]
Thanks for the reply.
Does this mean I have no choice but to run the factory firmware?
Do you have any suggestion for 1G router for openwrt in Japan?

**Reply by fakemanhk** [reply]
I don't know if stock firmware has the MAP-E support or not so I won't comment.
For OpenWrt based, you can look at something like Buffalo WSR-3200AX4S (Buffalo refurbished sales from time to time), Elecom WRC-X3200GST3, Asus RT-AX1800S (1800U), RT-AX4200, RT-AX6000, or get the GLINET MT6000 from AliExpress.

2 weeks ago in Akiba once of the junk shop having tons of old Elecom WRC-2533GST2 selling at 500 JPY (no power, you can add power which is about 1000) as well.

If you want to use existing RT-AC88U as AP then you can get NanoPi R2S/R4S from AliExpress

If you don't mind 802.11ac you can see if you are able to find something from used shop in Akiba according to following (check the latest support version 23.05.x)
https://openwrt.org/toh/hwdata/buffalo/start
https://openwrt.org/toh/hwdata/elecom/start
https://openwrt.org/toh/hwdata/nec/start

**Reply by twatwak** [reply]
Thanks!
What do you think about Netgear R7800? Its around 3-4k in mercari.

**Reply by LIII-XXII** [reply]
check the wiki notes for each model, it seems to have good support and enough flash memory for openwrt packages.
https://openwrt.org/toh/netgear/r7800
